macros/ec2_excessive_terminateinstances_mltk_input_filter.yml

```diff
@@ -0,0 +1,3 @@
+description: Use this macro to add additional filters for monitoring ec2 
+  runinstances [eg - userName != 'someUserNameExperiencingFalsePositives'].
```

Comment:
we should change it to terminate instances and change userName to src_user

Reply:
Done. Also fixed similar in ec2_excessive_runinstances_mltk_input_filter.yml. Thank you.